ios/RNSplashScreen.m

```diff
@@ -36,7 +36,15 @@ + (void)show {
 + (void)hide {
   // let's try to hide, even if showing == false, ...just in case
 
-  UIImageView *imageView = (UIImageView *)[UIApplication.sharedApplication.keyWindow.subviews.lastObject viewWithTag:RNSplashScreenOverlayTag];
+  NSArray *subviews = UIApplication.sharedApplication.keyWindow.subviews;
+  UIImageView *imageView = (UIImageView *)[subviews.lastObject viewWithTag:RNSplashScreenOverlayTag];
+
+  #ifdef DEBUG
+  if (imageView == nil) {
+    imageView = (UIImageView *)[subviews[subviews.count - 2] viewWithTag:RNSplashScreenOverlayTag];
```

Comment:
let's add a comment, otherwise it's a little mysterious
```suggestion
    // Allows the SplashScreen to be removed even when the perf monitor is the last subview.
    imageView = (UIImageView *)[subviews[subviews.count - 2] viewWithTag:RNSplashScreenOverlayTag];
```